ietf/nomcom/utils.py

```diff
@@ -536,20 +539,26 @@ def list_eligible_8713(date, base_qs=None):
     if not base_qs:
         base_qs = Person.objects.all()
     previous_five = previous_five_meetings(date)
-    return remove_disqualified(three_of_five_eligible(previous_five=previous_five, queryset=base_qs))
+    return remove_disqualified(three_of_five_eligible_8713(previous_five=previous_five, queryset=base_qs))
 
 def list_eligible_8788(date, base_qs=None):
     if not base_qs:
         base_qs = Person.objects.all()
     previous_five = Meeting.objects.filter(number__in=['102','103','104','105','106'])
-    return remove_disqualified(three_of_five_eligible(previous_five=previous_five, queryset=base_qs))
+    return remove_disqualified(three_of_five_eligible_8713(previous_five=previous_five, queryset=base_qs))
 
 def get_8989_eligibility_querysets(date, base_qs):
+    return get_threerule_eligibility_querysets(date, base_qs, three_of_five=three_of_five_eligible_8989)
+
+def get_8989bis_eligibility_querysets(date, base_qs):
+    return get_threerule_eligibility_querysets(date, base_qs, three_of_five=three_of_five_eligible_8989bis)
+
+def get_threerule_eligibility_querysets(date, base_qs, three_of_five):
```

Comment:
A comment (or type annotation) at least for the `three_of_five` parameter would be helpful. With all the similarly named methods and variables here it took me a while to sort out what was going on.